### PR TITLE
VACMS-4252_4253_4254_4255_4256_4257_4314 module removal and patch update

### PR DESCRIPTION
**Pull request by ethanteague**
## Description
Removes a bunch of unused modules, and moves a remote diff based patch to our local patches dir. All modules were previously disabled, so this just ensures that they aren't sucked in by composer. I checked for orphaned tables, unused variables / config in settings files, etc.
Also checked that the local patch applies cleanly.
See:
#4252 
#4253 
#4254 
#4255 
#4256 
#4257 
#4314 

## Testing done
Build, log checks, spot checks

## Screenshots
N/A

### Definition of Done

- [ ] Documentation has been updated, if applicable.
- [ ] Tests have been added if necessary.
- [ ] Automated tests have passed.
- [ ] Code Quality Tests have passed.
- [ ] Acceptance Criteria in related issue are met.
- [ ] Manual Code Review Approved.

### Select Team for PR review

- [x] `Core Application Team`
- [x] `Product Support Team`

### Is this PR blocked by another PR?

- [ ] `DO NOT MERGE`

### Does this PR need review from a Product Owner

- [ ] `Needs PO review`

### CMS user-facing annoucement

Is an announcement needed to let editors know of this change? 
- [ ] Yes, and it's written in issue ____ and queued for publication. 
  - [ ] Merge and ping @ rachel-kauff so she's ready to publish after deployment
- [ ] Yes, but it hasn't yet been written 
  - [ ] Don't merge yet -- ping @ rachel-kauff to prompt her to write and queue content
- [ ] No announcement is needed for this code change. 
  - [ ] Merge & carry on unburdened by announcements

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -355,7 +349,7 @@
                 "Fix revision uncaught type error by extending class.": "https://www.drupal.org/files/issues/2019-08-16/workflow_participants-revision-error-3075426-1.patch"
             },
             "traviscarden/behat-table-comparison": {
-                "Throw meaningful message for duplicate entry.": "https://patch-diff.githubusercontent.com/raw/TravisCarden/behat-table-comparison/pull/7.patch"
+                "Throw meaningful message for duplicate entry.": "patches/duplicate-row-message.patch"
```
Comment (swirtSJW) [inline composer.json:352]: Can you move the source of the patch into the comment so we can keep track of where it came from, or reference the issue?

**Review by swirtSJW — CHANGES_REQUESTED** [review]
Removals of modules all look good,  They check out as being uninstalled already.  

Just one historical issue (see comment below)

File: composer.json (hunk 1 above)
Comment (ethanteague) [reply composer.json]: Ah, good idea, @swirtSJW - glad you thought of that! Done :+1:

**Review by swirtSJW — APPROVED** [review]

**Comment by swirtSJW** [comment]
image

**Pull request merged**